README.md

```diff
@@ -43,6 +43,10 @@ await(ws.url("http://dns/url").get()).body == "http response"
 
 Add MockWS as test dependency in the `build.sbt`:
 
+* for Play 2.9.x:
```

Comment:
Shouldn't this be Play 3.0.x based on the things you change in this pull request? (Switching to pekko, setting `val playVersion = "3.0.0"` in `build.sbt`)

Reply:
Ah sorry, I totally missed #321, anyway you probably want to add entires for Play 3.0.0 then in the README :wink: Sorry for the noise